src/openklant/components/klantinteracties/api/tests/test_partijen.py

```diff
@@ -2067,158 +2080,164 @@ def test_invalid_choice_partij_identificator_code_objecttype(self):
-        # Overig no validation
-        url = reverse("klantinteracties:partijidentificator-list")
+    def test_invalid_update_partial(self):
+        # all partij_identificator fields required
         partij = PartijFactory.create()
+        list_url = reverse("klantinteracties:partijidentificator-list")
+        partij_identificator = PartijIdentificatorFactory.create(
+            partij=partij,
+            andere_partij_identificator="anderePartijIdentificator",
+            partij_identificator_code_objecttype="natuurlijk_persoon",
+            partij_identificator_code_soort_object_id="bsn",
+            partij_identificator_object_id="123456782",
+            partij_identificator_code_register="brp",
+        )
+
         data = {
-            "identificeerdePartij": {"uuid": str(partij.uuid)},
-            "anderePartijIdentificator": "anderePartijIdentificator",
+            "identificeerdePartij": None,
             "partijIdentificator": {
-                "codeObjecttype": "overig",
-                "codeSoortObjectId": "bsn",
-                "objectId": "296648875",
+                "codeObjecttype": "niet_natuurlijk_persoon",
+                "codeSoortObjectId": "rsin",
                 "codeRegister": "brp",
             },
         }
-        response = self.client.post(url, data)
+        detail_url = reverse(
+            "klantinteracties:partijidentificator-detail",
+            kwargs={"uuid": str(partij_identificator.uuid)},
+        )
+
+        response = self.client.patch(detail_url, data)
+
+        response = self.client.post(list_url, data)
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
         self.assertEqual(response.data["code"], "invalid")
         self.assertEqual(response.data["title"], "Invalid input.")
         self.assertEqual(
-            response.data["invalid_params"][0]["name"],
-            "partijIdentificatorCodeObjecttype",
+            response.data["invalid_params"][0]["name"], "partijIdentificator.objectId"
         )
-        self.assertEqual(response.data["invalid_params"][0]["code"], "invalid")
+        self.assertEqual(response.data["invalid_params"][0]["code"], "required")
         self.assertEqual(
-            response.data["invalid_params"][0]["reason"],
-            "voor `codeRegister` brp zijn alleen deze waarden toegestaan: ['natuurlijk_persoon']",
+            response.data["invalid_params"][0]["reason"], "Dit veld is vereist."
         )
+        self.assertEqual(PartijIdentificator.objects.all().count(), 1)
 
-    def test_valid_validation_partij_identificator(self):
-        # All validations pass
-        url = reverse("klantinteracties:partijidentificator-list")
-        partij = PartijFactory.create()
+
+class PartijIdentificatorUniquenessTests(APITestCase):
+    def setUp(self):
+        self.list_url = reverse("klantinteracties:partijidentificator-list")
+        self.partij = PartijFactory.create()
+        super().setUp()
```

Comment:
`setUp` is run before every test in the class, we can use `reverse_lazy` at the class level to avoid running it more than necessary 
```suggestion
    list_url = reverse_lazy("klantinteracties:partijidentificator-list")
    
    def setUp(self):
        self.partij = PartijFactory.create()
        super().setUp()
```